src/platform/tls_schannel.c

```diff
@@ -2123,6 +2123,45 @@ CxPlatTlsWriteDataToSchannel(
     }
 
     switch (SecStatus) {
+    case SEC_E_BUFFER_TOO_SMALL: {
+        //
+        // The output buffer for the TLS response is too small. We need to grow
+        // the buffer and try again.
+        //
+        QuicTraceLogConnInfo(
+            SchannelOutBufferTooSmall,
+            TlsContext->Connection,
+            "Increasing TLS output buffer size");
+        uint16_t NewBufferLength = State->BufferAllocLength << 1;
```

Comment:
Doesn't the assignment need a cast to prevent assigning a 4-byte int to a 2-byte int?

Reply:
well, it built, so I guess not.